settings.php

```diff
@@ -91,5 +91,15 @@
     $backgroundimage->set_updatedcallback('local_login_backgroundimage');
     $settings->add($backgroundimage);
 
+    // Load the contents of the minimal template file to be the default of the config.
+    $templatecontents = file_get_contents($CFG->dirroot . '/local/login/templates/login-minimal.mustache.default');
```

Comment:
again referencing this template path - would be good to have as a const or static func